src/framework/operations.hpp

```diff
@@ -771,7 +771,8 @@ Op json_to_op_roerror(const json_t &js) {
   op.name = "roerror";
   JSON::get_value(op.memory, "memory", js);
   JSON::get_value(op.registers, "register", js);
-  JSON::get_value(op.probs, "probabilities", js);
+  JSON::get_value(op.probs, "probabilities", js); // DEPRECIATED: Remove in 0.4
```

Comment:
Pretty sure you'll know better than me, but I'd swear that "depreciated" is used in financial-related context. it's more common to use "deprecate" here :)

Reply:
It's value has gone down since we included it